resin/models/application.py

```diff
@@ -488,3 +488,35 @@ def generate_provisioning_key(self, app_id):
             'POST',
             endpoint=self.settings.get('api_endpoint')
         )
+
+    def set_to_release(self, app_id, commit_id):
+        """
+        Set an application to a specific commit.
+        The commit will get updated on the next push unless rolling updates is disabled (there is a dedicated function for that which is resin.models.applicaion.disable_rolling_updates())
```

Comment:
NIT: `rolling updates is disabled` => `rolling updates are disabled`
UltraNIT: `dedicated function` => `dedicated method`, since it's a function under a namespace, but no strong opinion on this if it doesn't look Pythony